build.gradle

```diff
@@ -34,8 +34,6 @@ buildscript {
           'stdLib': [
```

Comment:
The stdlib is added automatically everywhere so none of these should be needed, realistically.